### ISSUE TITLE
Deleting one tag of a multi-platform image leaves session in a state where all tags show the same sha256 digest

### ISSUE THREAD

**Issue by plevart** [Joxit/docker-registry-ui#337]
Hi, I use this docker registry UI and I have an issue...

Latest release (2.5.2). Logging in (Basic Auth) shows the following list of tags (all images are mul;ti-platform, so I don't know whether this matters and can't compare behaviour with a single-platform image):

![image](https://github.com/Joxit/docker-registry-ui/assets/1580058/126d9bd2-ebbd-439d-b166-d321a89d3355)

Deleting one tag (for example, the oldest), works as expected:

![image](https://github.com/Joxit/docker-registry-ui/assets/1580058/7b4f6cd7-471a-4d15-b1a2-b59e04a87ad5)

But after that, when the deleted tag is gone from display, remaining tags strangely show the same SHA256:

![image](https://github.com/Joxit/docker-registry-ui/assets/1580058/fcf46139-862e-40c3-af56-cb217a5331d3)

When attempting to delete another tag in this state, the dialog suggests to delete all of them:

![image](https://github.com/Joxit/docker-registry-ui/assets/1580058/a67d759a-0262-4c62-85ad-a3eaa3bef6ef)

Reloading the browser page doesn't help. Only closing the tab and opening new one gives consistent view again:

![image](https://github.com/Joxit/docker-registry-ui/assets/1580058/0e078bc1-1025-496a-a616-a4337b586d61)

As can be seen, none of the image tags shown actually has the sha256 digest that is shown for all tags immediately after deleting one tag. It is totaly made-up from I don't know what.

My private docker registry configuration
```yml
version: 0.1
log:
  fields:
    service: registry
storage:
  cache:
    blobdescriptor: inmemory
  filesystem:
    rootdirectory: /data
  delete:
    enabled: true
auth:
  htpasswd:
    realm: basic-realm
    path: /etc/docker/registry/htpasswd
http:
  addr: :5000
  headers:
    X-Content-Type-Options: [nosniff]
    Access-Control-Allow-Origin: ['*']
    Access-Control-Allow-Credentials: [true]
    Access-Control-Allow-Headers: ['Authorization', 'Accept', 'Cache-Control']
    Access-Control-Allow-Methods: ['HEAD', 'GET', 'OPTIONS', 'DELETE']
    Access-Control-Expose-Headers: ['Docker-Content-Digest']
health:
  storagedriver:
    enabled: true
    interval: 10s
    threshold: 3
```

The Docker UI config (run as docker registry sidecar container in the same pod as docker registry - sharing network namespace):

```yml
# Listen interface.
listen_addr: 0.0.0.0:8000
# Base path of Docker Registry UI.
base_path: /

# Registry URL with schema and port.
registry_url: http://localhost:5000
# Verify TLS certificate when using https.
verify_tls: false

# Docker registry credentials.
# They need to have a full access to the registry.
# If token authentication service is enabled, it will be auto-discovered and those credentials
# will be used to obtain access tokens.
# When the registry_password_file entry is used, the password can be passed as a docker secret
# and read from file. This overides the registry_password entry.
registry_username: ...censored...
registry_password: ...censored...
# registry_password_file: /run/secrets/registry_password_file

# Event listener token.
# The same one should be configured on Docker registry as Authorization Bearer token.
event_listener_token: token
# Retention of records to keep.
event_retention_days: 7

# Event listener storage.
event_database_driver: sqlite3
event_database_location: data/registry_events.db
# event_database_driver: mysql
# event_database_location: user:password@tcp(localhost:3306)/docker_events

# You can disable event deletion on some hosts when you are running docker-registry on master-master or
# cluster setup to avoid deadlocks or replication break.
event_deletion_enabled: true

# Cache refresh interval in minutes.
# How long to cache repository list and tag counts.
cache_refresh_interval: 10

# If all users can view the event log. If set to false, then only admins listed below.
anyone_can_view_events: true
# If all users can delete tags. If set to false, then only admins listed below.
anyone_can_delete: true
# Users allowed to delete tags.
# This should be sent via X-WEBAUTH-USER header from your proxy.
admins: []

# Debug mode. Affects only templates.
debug: true

# How many days to keep tags but also keep the minimal count provided no matter how old.
purge_tags_keep_days: 14
purge_tags_keep_count: 5

# Keep tags matching regexp no matter how old, e.g. '^latest$'
# Empty string disables this feature.
purge_tags_keep_regexp: ''

# Keep tags listed in the file no matter how old.
# File format is JSON: {"repo1": ["tag1", "tag2"], "repoX": ["tagX"]}
# Empty string disables this feature.
purge_tags_keep_from_file: ''

# Enable built-in cron to schedule purging tags in server mode.
# Empty string disables this feature.
# Example: '25 54 17 * * *' will run it at 17:54:25 daily.
# Note, the cron schedule format includes seconds! See https://godoc.org/github.com/robfig/cron
purge_tags_schedule: ''
```

## Expected behavior

Without this bug

## System information

- OS: [OCI image executed on k8s]
- Browser:
    - Name: Brave
    - Version: 1.58.131 Chromium: 117.0.5938.92
- Docker registry:
    - Version: 2.8.2
- Docker registry UI:
    - Version: 2.5.2
    - Server: k8s cluster
    - Docker version: (not docker - cri-o 1.26)
    - Docker registry ui tag: 2.5.2
    - OS/Arch: linux/arm64
    - Tools: k8s 1.26.0

**Comment by tranlybuu** [comment]
i got the same issue when i want to delete an image
<img width="1440" alt="Screenshot 2023-10-04 at 10 52 44" src="https://github.com/Joxit/docker-registry-ui/assets/70121634/48866b1b-ebd9-45e7-b3e0-f62b90ec961e">

**Comment by Joxit** [comment]
Hi, thank you for using my project and posting issues :smile: 

@plevart  Your bug has been confirmed. It seems that it's not only when deleting images but refreshing too. Should be fixed in [2.5.4](https://github.com/Joxit/docker-registry-ui/releases/tag/2.5.4).

@tranlybuu Your question looks like one of the [FAQ](https://github.com/Joxit/docker-registry-ui#faq) questions, you pushed the same image with differents tags right ? So they are sharing the same SHA.
> * Why when I delete one tag, all tags with the same SHA are deleted ?
>   * This a docker registry API limitation, there is only one way to [delete images with tag](https://docs.docker.com/registry/spec/api/#deleting-an-image), it's by its name and its manifest (it's a sha of the content). So when you delete a tag, this will delete all tags of this image with the same SHA/manifest.

**Comment by plevart** [comment]
Thanks @Joxit , I can confirm that with 2.5.4. it works as expected.
